docker/serve/Dockerfile

```diff
@@ -4,7 +4,8 @@ ARG CUDNN="7"
 FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-devel
 
 ARG MMCV="1.4.4"
-ARG MMSEG="0.21.0"
+ARG MMSEG="0.22.0"
+ARG MMCLS="0.21.0"
```

Comment:
might remove this line, I remember when installing mmseg, it will call `requirements/runtime.txt` and mmcls will be installed

and in fact add an `ARG` in this line that will not install mmcls, you may refer Line 31 in this file fo mmcv installation